drivers/clock_control/stm32_ll_clock.c

```diff
@@ -223,7 +223,9 @@ static int stm32_clock_control_init(struct device *dev)
 
 	/* Disable other clocks */
 	LL_RCC_HSE_Disable();
+#if !defined(CONFIG_SOC_SERIES_STM32F7X)
 	LL_RCC_MSI_Disable();
+#endif
```

Comment:
Please define LL_RCC_MSI_Disable with do nothing in stm32f7x_ll_clock.c, as done for F4
This helps minimizing the #ifdef stuff in the main file